parentheses/service/service.go

```diff
@@ -0,0 +1,41 @@
+// Package service implements a parentheses web service that
+// generates a random sequence of parentheses of the given length.
+package service
+
+import (
+	"ita/parentheses/brackets"
```

Comment:
You'll need to import this by the mod name and package 
`"github.com/Konstantsiy/go-internship/parentheses/brackets"`